pom.xml

```diff
@@ -112,8 +112,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
I would just leave 1.7, until there is a good reason to be 1.8.

that way we would move the codebase to 1.8 on purpose, where there is the API need.